python-checks/src/main/resources/org/sonar/l10n/py/rules/python/S2712.json

```diff
@@ -1,7 +1,7 @@
 {
   "title": "\"return\" and \"yield\" should not be used in the same function",
   "type": "BUG",
-  "status": "ready",
+  "status": "deprecated",
```

Comment:
I don't think this should be deprecated, as it is still relevant for Python 2 developers. Note that this rule is not on SonarWay anyway.